config/generic-steval-3dp001v1.cfg

```diff
@@ -0,0 +1,327 @@
+# This file contains common pin mappings for STEVAL-3DP001V1 board.
+# to use this config, the firmware should be compiled for the STM32F4
```

Comment:
This will need to be expanded on, it has no mention of how you should set the make menuconfig, clocks, speeds, bootloader offsets etc. and then the method of flashing this file.